src/ert/_c_wrappers/enkf/data/gen_data.py

```diff
@@ -31,14 +28,6 @@ def free(self):
     def __repr__(self):
         return f"GenData(len = {len(self)}) {self._ad_str()}"
 
-    def export(self, file_name, file_format_type, fortio):
```

Comment:
My IDE is telling me this export function is used in `src/ert/gui/tools/export/export_tool.py` not sure if that is true or not but worth double checking.

Reply:
thank you, gonna investigate!

ya it's a bit so-so, i didn't go that deep, figured its gui so it's fine, checked it out, it defines an export function, and that one calls another export funtion - on something called `__export_widget()` - i just assumed that this widget is something QT-y, and not related to a gen data thingy ^^

and indeed, now looking a bit deeper, the `__export_widget` gets assigned a `ref` of something that inherits from a QT widget, and has an `export` function defined, which does not do anything with gen data.

this is why i cannot trust IDEs anymore ; )